__tests__/app.test.js

```diff
@@ -67,4 +66,47 @@ describe('GET /api/reviews', () => {
         descending: true
       })
     });
-  });
\ No newline at end of file
+  })
+});
+
+describe('GET /api/reviews/:review_id', () => {
+  test('should return an array of comments for the provided review_id', () => {
+    return request(app)
+      .get("/api/reviews/1/comments")
+      .expect(200)
+      .then(({ body }) => {
+        const { comments } = body;
+          const expected = {          
+          comment_id: expect.any(Number),
+          review_id: expect.any(Number),
+          created_at: expect.any(String),
+          votes: expect.any(Number),
+          author: expect.any(String),
+          body: expect.any(String)
+      }
+      comments.forEach(comment => {
+        expect(comment).toEqual(expect.objectContaining(expected))
+      })
+      expect(comments.length).toBe(3);
+      expect(comments).toBeSortedBy('created_at', {
+        descending: true
+      })
+      });
+  });
```

Comment:
You need to include another 200 test for an existing article but for one that has no comments.